app/src/main/java/org/covidwatch/android/data/contactevent/ContactEventFetcher.kt

```diff
@@ -15,6 +15,16 @@ interface ContactEventFetcher {
      * @return The resulting contact events.
      * @exception Exception Throws an exception on error.
      */
-    fun fetch(timeWindow: ClosedRange<Date>,
-              resultCb: List<String>.(infectionState: InfectionState) -> Unit)
+    fun fetch(timeWindow: ClosedRange<Date>)
+
+    /**
+     * Start listening for contact events from the backend.
+     * @exception Exception Throws an exception on error.
+     */
+    fun startListening()
+
+    /**
+     * Stop listening for contact events from the backend.
+     */
+    fun stopListening()
```

Comment:
we can start and stop listening in [MainViewModel.kt](https://github.com/covid19risk/covidwatch-android/blob/bb600568fddc8062668af8192257cd2f3ce0496c/app/src/main/java/org/covidwatch/android/presentation/MainViewModel.kt) from my [draft PR](https://github.com/covid19risk/covidwatch-android/pull/93)